bin/collator/src/command.rs

```diff
@@ -44,7 +44,12 @@ use sp_runtime::traits::Block as BlockT;
 use std::net::SocketAddr;
 
 #[cfg(feature = "frame-benchmarking")]
-use frame_benchmarking_cli::{BenchmarkCmd, SUBSTRATE_REFERENCE_HARDWARE};
+use frame_benchmarking_cli::{BenchmarkCmd, ExtrinsicFactory, SUBSTRATE_REFERENCE_HARDWARE};
+
+#[cfg(feature = "runtime-benchmarks")]
+use crate::benchmarking::*;
+#[cfg(feature = "runtime-benchmarks")]
+use sp_keyring::Sr25519Keyring;
```

Comment:
Perhaps these imports can just be added under the `Benchmark` cmd below?

Would make the code look cleaner.

Reply:
Yes, thank you for suggestion! that's much cleaner.